### DISCUSSION TITLE
Purchase more data packs to restore access. LFS

### DISCUSSION THREAD

**Discussion by Evdokimov27** [community/community#21965] [New to GitHub]
<p>I am a beginner game developer from Russia, my team and I are trying to somehow exist at this time, but everyone considers us enemies of humanity, they don’t even let me buy the necessary datapacks for LFS to work, how can I work if they don’t give me this opportunity?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there and welcome to the Community Forum <a class="mention" href="/evdokimov27">@Evdokimov27</a>! For account-specific questions like this, I recommend contacting our <a href="https://support.github.com/contact">Support team</a> as they’ll be best able to assist here.</p>
